include/libswiftnav/signal.h

```diff
@@ -37,16 +58,14 @@ enum band {
 
 typedef struct __attribute__((packed)) {
   u16 sat;
-  u8 band;
-  u8 constellation;
+  enum code code;
+  u8 reserved;
```

Comment:
@cbeighley do you want to comment on this since you voted for to keep reserved field?